Source/WebCore/platform/graphics/gstreamer/mse/AppendPipeline.cpp

```diff
@@ -246,17 +246,12 @@ void AppendPipeline::handleErrorConditionFromStreamingThread()
 {
     ASSERT(!isMainThread());
     // Notify the main thread that the append has a decode error.
-    auto response = m_taskQueue.enqueueTaskAndWait<AbortableTaskQueue::Void>([this]() {
+    m_taskQueue.enqueueTaskAndWait<AbortableTaskQueue::Void>([this]() {
         m_errorReceived = true;
         // appendParsingFailed() will cause resetParserState() to be called.
```

Comment:
I'm wondering if after the changes you mention this comment is still true. I hope it is.

Reply:
It not being true is the reason the assert fails, actually.

Reply:
Remove or update this comment and otherwise LGTM. If the missing `resetParserState()` call ends up botching error handling that's a separate bug anyway.